### DISCUSSION TITLE
package.json里面配置是否正确

### DISCUSSION THREAD

**Discussion by wangweifine** [coder-hxl/x-crawl#116] [General]
"dependencies": {
    "chalk": "5.3.0",
    "https-proxy-agent": "^7.0.4",
    "openai": "^4.33.0",
    "ora": "^8.0.1",
    "puppeteer": "22.5.0",
    "x-crawl": "link:"
  },  这个x-crawl: 为link: 对吗，如果不对，应该写什么

**Comment by coder-hxl** [comment]
这个是没问题的
